toolkit/offchain/src/plutus_script.rs

```diff
@@ -0,0 +1,111 @@
+use anyhow::{anyhow, Context};
+use cardano_serialization_lib::{Address, LanguageKind, NetworkIdKind, PlutusData};
+use plutus::ToDatum;
+use sidechain_domain::{MainchainAddressHash, PolicyId};
+use uplc::ast::{DeBruijn, Program};
+
+use crate::{csl::*, untyped_plutus::*};
+
+/// Wraps a Plutus script cbor
+pub struct PlutusScript {
+	pub bytes: Vec<u8>,
+	pub language: LanguageKind,
+}
+
+impl PlutusScript {
+	pub fn from_cbor(cbor: &[u8], language: LanguageKind) -> Self {
+		Self { bytes: cbor.into(), language }
+	}
+
+	// This function is needed to create [PlutusScript] from scripts in [raw_scripts],
```

Comment:
Cosmetic: if this was `///` comment and you have used backticks, then I think it would be a proper link